main.js

```diff
@@ -0,0 +1,150 @@
+var startingPeg = ["5", "4", "3", "2", "1"];
+var startingArr1 = [];
+var startingArr2 = [];
+var startingArr3 = [];
+var originalPeg;
+var playingBoard;
+
+var randomizeArrays = function() {
+    randomNum = Math.floor(Math.random() * 3);
+
+    if (randomNum === 0) {
+        originalPeg = 1;
+        startingArr1 = startingPeg.map(function (element){
```

Comment:
Typically you'd use `map` to mutate data. The way you're using it here is essentially copying the array, which you could do with the spread operator, for example (`[...]`)